Cookbook/README.md

```diff
@@ -0,0 +1,38 @@
+iOS Cookbook 👩‍🍳
+ ====================================
+
+## I am here because I want to ...
+
+##### ... learn about our architecture 🏠
+
+* [Implementing features with ReactiveFeedback](http://ilya.puchka.me/implementing-features-with-reactivefeedback/)
+* [Global `Current` struct to hold global dependencies](./Proposals/ControlTheWorld.md)
+* [Posting Analytics Events to Snowplow](./Technical-Documents/SnowplowHowTo.md)
+* [Use builders as functions](./Technical-Documents/BuildersToFunctions.md)
+
+##### ... build my feature with Bento 🍱
+
+* [Generalized Screen Lifecycle Observation](./Proposals/ScreenLifecycleObservation.md)
+* [Design Library](./Technical-Documents/DesignLibrary.md)
+
+##### ... learn how the iOS team works 🛠
+
+* [Code Owners - Restoring developer's efficiency](./Proposals/CODEOWNERS.md)
+* [iOS Team style guide](./Style-guide/)
+* [New hires checklist](./Technical-Documents/NewHiresChecklist.md)
+* [How to make a proposal](./Proposals/Template_Proposal.md)
```

Comment:
This document might probably also need to be updated to mention:
 - the new path under which to create new Proposals (Cookbook/Proposals)
 - that one should add an entry in this Cookbook document at some point to reference new proposals being adopted